src/modules/visits/dto/visits.dto.ts

```diff
@@ -155,6 +173,17 @@ export class VisitsQueryDto {
 	})
 	visitorSpaceSlug?: string;
 
+	@IsString()
+	@IsEnum(VisitAccessType)
+	@IsOptional()
+	@ApiPropertyOptional({
+		type: String,
+		description: 'Access type of the visit',
+		enum: VisitAccessType,
+		default: VisitAccessType.FULL,
```

Comment:
no default, since we want to get all visit requests if no filter value was provided for accessType
```suggestion
		default: undefined,
```